scripts/verify_hutch

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# function that takes in a string and verifies its a known hutch
+
+usage(){
+cat << EOF
+usage: ${BASH_SOURCE[0]} <ioc>|<pv> [option]
+
+Script that returns confirmation if the passed hutch is a known/existing hutch
+
+EOF
+}
+
+#hutches=['tmo','txi','rix','xpp','xcs','mfx','cxi','mec', 'ued', 'det', 'lfe','kfe','tst', 'las', 'hpl']
+
+foundHutch=False
+HUTCH=${1,,}
```

Comment:
I recommend lowercase variable name for script-specific variables and just to match that its a lower case hutch name
```suggestion
hutch=${1,,}
```